### PR TITLE
Define "high-entropy client hint"

### LINKED ISSUE #126: Define "high entropy" hint
We don't define this anywhere (AFAICT).

We could say something like "a high entropy hint is any hint that's not in https://wicg.github.io/client-hints-infrastructure/#low-entropy-hint-table".

### PR DESCRIPTION
**Pull request by miketaylr**
PTAL @yoavweiss 

(we could also add a note to clarify this means all hints are high-entropy by default, but maybe that's obvious)

Fixes #126

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/client-hints-infrastructure/pull/127.html" title="Last updated on Sep 19, 2022, 1:09 PM UTC (3b2a22a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/127/63db401...miketaylr:3b2a22a.html" title="Last updated on Sep 19, 2022, 1:09 PM UTC (3b2a22a)">Diff</a>

### CODE REVIEW

**Review by arichiv — APPROVED** [review]
Approved, will wait 24hrs if anyone else has comment before merge

File: index.bs
Comment (arichiv) [inline index.bs:326]: This is a little circular, is there any other possible reference?

**Review by tomayac — APPROVED** [review]
LGTM

File: index.bs
Comment (miketaylr) [reply index.bs:326]: Any suggestions?

Comment (arichiv) [reply index.bs:326]: Not seeing anything obvious https://github.com/search?p=1&q=org%3AWICG+entropy&type=Code

Maybe it's resolved by saying "defines hints *safe to send by default* due to their low amounts of entropy." That way there's some indication why it even matters.

**Review by yoavweiss — APPROVED** [review]
LGTM % considering switching to @arichiv's version..

File: index.bs
Comment (miketaylr) [reply index.bs:326]: Good suggestion, thanks.

**Comment by miketaylr** [comment]
Thanks all

**Pull request merged**
